packages/block-library/src/navigation/edit/index.js

```diff
@@ -283,7 +297,22 @@ function Navigation( {
 		hasResolvedCanUserCreateNavigationMenu,
 	} = useNavigationMenu( ref );
 
+	// Attempt to retrieve and prioritize any existing navigation menu
```

Comment:
Looks like we lost the longer comment. I think we should restore it:
```suggestion
	// Attempt to retrieve and prioritize any existing navigation menu unless
	// a specific ref is allocated or the user is explicitly creating a new menu. The aim is
	// for the block to "just work" from a user perspective using existing data.
```

Reply:
😄 I thought [you suggested to remove it](https://github.com/WordPress/gutenberg/pull/39880#discussion_r840400621)?

Reply:
Maybe I removed the wrong one? 👀

Reply:
Restored.